easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'FFTW'
+version = '3.3.6'
+
+homepage = 'http://www.fftw.org'
+description = """FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
+ in one or more dimensions, of arbitrary input size, and of both real and complex data."""
+
+toolchain = {'name': 'gimpi', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = ['fftw-%(version)s-pl2.tar.gz']
+checksums = ['a5de35c5c824a78a058ca54278c706cdf3d4abba1c56b63531c2cb05f5d57da2']
+
+skipsteps = ['test']
+#runtest = 'check'
```

Comment:
@akesandgren Why is this here?

Reply:
oops, forgot to remove it. Had problems with the mpi tests on my laptop for some reason.